zhaquirks/schneiderelectric/__init__.py

```diff
@@ -0,0 +1,135 @@
+"""Quirks implementations for Schneider Electric devices."""
+from typing import Any, Coroutine, Dict, Union
+
+from zigpy import types as t
+from zigpy.quirks import CustomCluster
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import Basic
+from zigpy.zcl.foundation import ZCLAttributeDef
+
+SE_MANUF_NAME = "Schneider Electric"
+SE_MANUF_ID = 4190
+
+
+class SEBasic(CustomCluster, Basic):
+    """Schneider Electric manufacturer specific Basic cluster."""
+
+    attributes: Dict[int, ZCLAttributeDef] = Basic.attributes.copy()
+
+    attributes.update(
+        {
+            0xE001: (
+                "se_sw_build_id",
+                t.CharacterString,
+                True,
+            ),  # value: "002.004.016 R
```

Comment:
Ideally, we already want to use the new zigpy-style `AttributeDefs` definitions, like this:
https://github.com/zigpy/zha-device-handlers/blob/ab0d248defda933019c98ddab451b5969fe99c16/zhaquirks/legrand/cable_outlet.py#L42-L59

Reply:
Ok, I’ve updated the code.